test/e2e/framework/testcontext.go

```diff
@@ -33,10 +33,20 @@ type ExposeOptions struct {
 	ClientsBroadcastAddressType scyllav1.BroadcastAddressType
 }
 
+type ObjectStorageType string
+
+const (
+	ObjectStorageTypeNone ObjectStorageType = "none"
+	ObjectStorageTypeGCS  ObjectStorageType = "gcs"
```

Comment:
nit
```suggestion
	ObjectStorageTypeNone ObjectStorageType = "None"
	ObjectStorageTypeGCS  ObjectStorageType = "GCS"
```

Reply:
The idea was that the type simultaneously functions as a prefix for scylla-manager location, since it takes `<backend>:[<dc>:]:<bucket-name>`, but I guess I could always add a func returning a prefix string for a given type. Wdyt?

Reply:
I'd not couple them. It would be nice to keep the constant case rules consistent and a conversion switch might be appropriate - although maybe `ToLower()` can replace the switch

Reply:
added an extra method with switch